retrofit/src/main/java/retrofit2/Retrofit.java

```diff
@@ -542,6 +542,11 @@ public Builder callbackExecutor(Executor executor) {
       return this;
     }
 
+    /** Returns a modifiable list of call adapter factories. */
```

Comment:
For both `converterFactories()` and `adapterFactories()`, is "modifiable" sufficiently descriptive?
That is, it's very different from returning `new ArrayList<>(adapterFactories)` which is also mutable.
This exposes the builder's list to modify the builder's state.

Reply:
This language is taken from `OkHttpClient.interceptors()` and `OkHttpClient.networkInterceptors()`: 

"Returns a modifiable list of interceptors ..."

Reply:
Ah, cool, precedence. Sgtm!